docs/02-quick-start/01-installation/02-kubernetes.md

```diff
@@ -1,7 +1,107 @@
 ---
 sidebar_label: Kubernetes
 ---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
 
 # Try Ozone With Kubernetes
 
-**TODO:** File a subtask under [HDDS-9856](https://issues.apache.org/jira/browse/HDDS-9856) and complete this page or section.
+## Minikube
+
+**Requirements:**
+
+* Working minikube setup
+* kubectl
+
+`kubernetes/examples` folder of the Ozone distribution contains Kubernetes deployment resource files for multiple use cases. By default the Kubernetes resource files are configured to use *apache/ozone* image from the Docker Hub.
+
+To deploy it to minikube, use the minikube configuration set:
+
+``` bash
+cd kubernetes/examples/minikube
+kubectl apply -k .
+```
+
+And you can check the results with
+
+``` bash
+kubectl get pod
+```
+
+Note: the `kubernetes/examples/minikube` resource set is optimized for minikube usage:
```

Comment:
For callouts like this we should docusaurus admonitions as described in the [contributing guide](https://github.com/apache/ozone-site/blob/HDDS-9225-website-v2/CONTRIBUTING.md#page-format).